src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -36,6 +43,10 @@ export interface DatasourceOverwrite {
   url: string
 }
 
+const logger = (...args) => {
```

Comment:
What is this for?

Reply:
Maybe it was for debug, I found it used in the file, should this be removed maybe?

Reply:
Yes, this is for debug and I would like to keep it around until I'm done with the error links